src/national.py

```diff
@@ -36,76 +35,43 @@
 def get_national_forecast(
     request: Request,
     session: Session = Depends(get_session),
-    historic: Optional[bool] = False,
-    only_forecast_values: Optional[bool] = False,
     forecast_horizon_minutes: Optional[int] = None,
     user: Auth0User = Security(get_user()),
 ) -> Union[Forecast, List[ForecastValue]]:
     """Get the National Forecast
 
-    This route aggregrates data from all GSP forecasts and creates an 8-hour solar energy
-    generation forecast  in 30-minute interval for all of GB.
+    This route aggregrates data from all GSP forecasts in Great Britain,
+    creating a national, 8-hour,
+    solar generation forecast in 30-minute intervals.
+    The _forecast_horizon_minutes_ parameter allows
+    a user to query for a forecast closer than 8 hours to the target time.
 
```

Comment:
I wonder if we should update this. 
- Get ride of "aggregrates data" bit as we now have a separate forecast that just does national
- We should say something like. This routes returns the most recent forecast for each target_time. 
- The _forecast_horizon_minutes_ parameters allows the user to query a forecast that is made this number of minutes before the target time. For example ... (same as below)

Reply:
![Screenshot 2023-06-27 at 18 35 31](https://github.com/openclimatefix/uk-pv-national-gsp-api/assets/86949265/2fcdc364-d8e4-4f85-9d31-5d34f9e86e0b)